### PR TITLE
[context.trakt.watched] 1.0.3

### PR DESCRIPTION
**Pull request by Razzeee**

### CODE REVIEW

File: context.trakt.watched/addon.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  </requires>
+    <extension point="kodi.context.item" library="addon.py">		
+      <item>
+        <label>32000</label>
+        <visible>String.IsEqual(ListItem.dbtype,movie) | String.IsEqual(ListItem.dbtype,tvshow) | String.IsEqual(ListItem.dbtype,season) | String.IsEqual(ListItem.dbtype,episode)</visible>
+      </item>
+    </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="de">Trakt-Schaltfläche für Gesehene zum Kontextmenü in der Bibliothek hinzufügen.</summary>
+    <summary lang="en">Add Trakt watched button to context menu in library</summary>
```
Comment (romanvm) [inline context.trakt.watched/addon.xml]: Haven't noticed at first. Default language code is `en_GB`

File: context.trakt.watched/addon.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="context.trakt.watched" name="Trakt - Watched button" version="1.0.3" provider-name="Razzeee">
+  <requires>
+    <import addon="xbmc.python" version="2.20.0"/>
```
Comment (romanvm) [inline context.trakt.watched/addon.xml]: 2.25.0

**Comment by romanvm** [comment]
Please check python API version and the default language code in all your submissions. They should be `2.25.0` and `en_GB` respectively.

File: context.trakt.watched/addon.xml (hunk 1 above)
Comment (MartijnKaijser) [reply context.trakt.watched/addon.xml]: same for de_DE :)

**Comment by MartijnKaijser** [comment]
Should also mention that you should move to new language folder layout ;)
Also icon.png should have solid background

**Comment by Razzeee** [comment]
Updated

**Pull request merged**